### PR TITLE
Enhancements

### LINKED ISSUE #36: Add version command

### LINKED ISSUE #37: Add clean script for development

### PR DESCRIPTION
**Pull request by altafan**
* Pretty start logs
* move `.env` to `~/.nigiri` instead of being floating into fllesystem
* add `scripts/clean` for development. Closes #37 
* add `--version` flag. Closes #36 

NOTE: paths must be changed before merging

### CODE REVIEW

File: cli/cmd/start.go [hunk 1]
```diff
@@ -312,3 +329,20 @@ func mergeComposeEnvFiles(rawJSON []byte) map[string]map[string]int {
 
 	return mergedPorts
 }
+
+// loadEnv is used as workaround to load datadir/.env in compose since
+// the flag --project-directory does not seem to work, but if running
+// docker-compose directly from terminal
+// https://forums.docker.com/t/running-docker-compose-from-golang/75006
```
Comment (tiero) [inline cli/cmd/start.go]: remove this not relevant

File: scripts/clean [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+esac
+
+case $(uname -m) in
+  amd64) ARCH="amd64";;
+  x86_64) ARCH="amd64";;
+  *) echo "Architecture $ARCH not supported"; exit 1;;
+esac
+
+./build/nigiri-$OS-$ARCH stop --delete &>/dev/null
+rm -rf build vendor ~/.nigiri
```
Comment (tiero) [inline scripts/clean]: .env?

Comment (altafan) [reply scripts/clean]: env is inside the datadir

**Comment by tiero** [comment]
@altafan add clean script in *directions*

**Comment by altafan** [comment]
fixed

**Pull request merged**